packages/sdk-js/src/index.ts

```diff
@@ -27,6 +27,7 @@ import {
   GetWorkflowsRequest,
   DEFAULT_LIMIT,
   ListSecretResponse,
+  ListSecretsResponse,
```

Comment:
The two functions seem to be duplicates
 ListSecretResponse,
 ListSecretsResponse,